src/gimelstudio/core/node/property.py

```diff
@@ -355,3 +355,23 @@ def WidgetEventY(self, event):
 
     def WidgetEventZ(self, event):
         self.SetValue((self.value[0], self.value[1], event.value))
+
+
+class ActionProp(Property):
+    """
+    Allows the user to click a button to perform an action
+    """
+    def __init__(self, idname, default="", label="", flat=False, action=None, visible=True):
```

Comment:
Could you add a separate param (e.g: ``btn_lbl``) for the button label so that the fold panel label can be different than the button label if we so desire? The ``label`` param should define the fold panel label.

Reply:
> The label param should define the fold panel label

To be honest, I disagree with this. I think the fold panel label should be called ``fpb_label`` and the button label ``label``.